opal/corelib/enumerable.rb

```diff
@@ -177,6 +177,10 @@ def count(object = undefined, &block)
   end
 
   def cycle(n = nil, &block)
+    if `arguments.length > 1`
```

Comment:
@meh But without this `if` an associated spec fails. Is it a problem with running build on CI (that we do not pass some extra options to compiler) or am I missing something?